erpnext/crm/doctype/lead/lead.json

```diff
@@ -454,13 +456,21 @@
    "fieldtype": "Link",
    "label": "Print Language",
    "options": "Language"
+  },
+  {
+   "fieldname": "column_break_17",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "column_break_22",
+   "fieldtype": "Column Break"
   }
  ],
  "icon": "fa fa-user",
  "idx": 5,
  "image_field": "image",
  "links": [],
- "modified": "2021-01-06 19:39:58.748978",
+ "modified": "2021-05-27 16:42:55.453326",
```

Comment:
Undo the changes in this file since you have already refactored the lead form in another PR.

Reply:
Done